crates/hotshot-signature-key/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "hotshot-signature-key"
+description = "Signature instantiations"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+
+[dependencies]
+bincode = { workspace = true }
+bitvec = { workspace = true }
+blake3 = { workspace = true }
+custom_debug = { workspace = true }
+ethereum-types = { workspace = true }
+hotshot-qc = { path = "../../crates/hotshot-qc" }
+hotshot-types = { path = "../../types" }
+hotshot-utils = { path = "../../utils" }
+jf-primitives = { workspace = true }
+rand = { workspace = true }
+rand_chacha = { workspace = true }
+serde = { workspace = true }
+tracing = { workspace = true }
+typenum = { workspace = true }
+
+[features]
+# default = ["parallel"]
+# std = ["ark-std/std"]
+# parallel = ["jf-primitives/parallel", "jf-utils/parallel"]
+full-ci = ["demo"]  # needed only because `just test_*` needs `demo` code
+tokio-ci = ["demo"] # needed only because `just test_*` needs `demo` code
+demo = []
```

Comment:
someone know better how we should declare flags in this repo vet this please? 

plus could we write a section in CONTRIBUTING.md to include a simple explanation of these feature flag and how to use them?